src/style.css

```diff
@@ -0,0 +1,126 @@
+* {
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+h3 {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+:root {
+  --btn-color: #00acdc;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.product__img {
+  align-items: center;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.product__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  padding-bottom: 4px;
+  padding-top: 40px;
+}
+
+.product__code {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin-bottom: 17px;
+}
+
+.product__rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(-N+4) {
```

Comment:
```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
```

Reply:
this point not fixed
![Screenshot from 2023-04-15 16-52-32](https://user-images.githubusercontent.com/33029864/232228532-e41bad7a-c0ab-411e-9855-7e3641f4aee2.png)
it means that should be added styles with all stars--{n} modifiers